parity/main.rs

```diff
@@ -186,6 +186,13 @@ fn execute_client(conf: Configuration, spec: Spec, client_config: ClientConfig)
 	let net_settings = conf.net_settings(&spec);
 	let sync_config = conf.sync_config(&spec);
 
+	// Create and display a new token for UIs.
+	if conf.args.flag_signer && !conf.args.flag_no_token {
```

Comment:
as long as people have "don't remember third party data" checked, like me, then it'll still grow, only also inconvenience the user while it does...